getters.py

```diff
@@ -56,14 +56,14 @@ def get_message():
     functions.display_msg(msg)
 
     continue_flag = input('''
-Do you wish to continue?
+Do you wish to continue the message?
 Enter y if yes and n if no
     ''')
 
     if continue_flag == 'y':
-        return msg
-    elif continue_flag == 'n':
         get_message()
+    elif continue_flag == 'n':
+        return msg
```

Comment:
In this case if the user entered "y" it will reject the message and ask him to enter the message again, that's not desirable.